docs/getting-started/start-app.md

```diff
@@ -23,6 +23,11 @@ If you run into issues while trying to run `bin/setup` and the error message
 isn't helpful, try running `bin/rails s -p 3000`. For example, you may need to
 `yarn install` before starting the app.
 
+If Sidekiq is producing errors similar to
+`No such file or directory - [SOME FILE]`, you may need to start Sidekiq by
```

Comment:
I'm not quite sure this is the actual reason. In the past I mostly noticed this when Redis was recently updated.

Reply:
To the best of my (limited) knowledge, this was different. I hadn't been able to get a successful run of `bin/startup` on my machine at all, and I do not think I stopped or restarted the Redis server in between hitting this failure and resolving it with this independent Sidekiq startup.